app/validation/validation_result.py

```diff
@@ -1,9 +1,26 @@
 class ValidationResult(object):
+    def __init__(self, is_valid=False):
+        self.is_valid = is_valid
+        self.errors = {}
+        self.warnings = {}
+
     def is_valid(self):
-        raise NotImplementedError()
+        return self.is_valid
 
     def get_errors(self):
```

Comment:
The validator classes will be refactored once lewis's changes have been merged in